app/src/main/AndroidManifest.xml

```diff
@@ -35,6 +37,42 @@
         <activity android:name=".FAQs" />
         <activity android:name=".Workshops" />
         <activity android:name=".Sponsors" />
+
+        <service
+            android:name=".MyFirebaseMessagingService">
+            <intent-filter>
+                <action android:name="com.google.firebase.MESSAGING_EVENT" />
+            </intent-filter>
+        </service>
+
+        <service
+            android:name=".MyFirebaseInstanceIdService">
+            <intent-filter>
+                <action android:name="com.google.firebase.INSTANCE_ID_EVENT"/>
+            </intent-filter>
+        </service>
+
+        <!-- Set custom default icon. This is used when no icon is set for incoming notification messages.
```

Comment:
Change this to a `TODO: decide what icon we want here` so that we can keep track of this. We may want an icon for notifications but I don't think that is a decision we are making for this PR; so let's leave a `TODO` so that we can come back to it.